### ISSUE TITLE
Chapter 13, exersize 7 mistake

### ISSUE THREAD

**Issue by lrunaways** [ageron/handson-ml#304]
We make dropout layer fc1_drop:
```
with tf.name_scope("fc1"):
    fc1 = tf.layers.dense(pool3_flat_drop, n_fc1, activation=tf.nn.relu, name="fc1")
    fc1_drop = tf.layers.dropout(fc1, fc1_dropout_rate, training=training)
```
But use fc_1 building output dense layer:
```
with tf.name_scope("output"):
    logits = tf.layers.dense(fc1, n_outputs, name="output")
    Y_proba = tf.nn.softmax(logits, name="Y_proba")
```
Should be
```
with tf.name_scope("output"):
    logits = tf.layers.dense(fc1_drop, n_outputs, name="output")
    Y_proba = tf.nn.softmax(logits, name="Y_proba")
```

**Comment by ageron** [comment]
Thanks @lrunaways ,
I'm currently traveling, hence the long time to respond. I'll try to fix this asap.
Cheers,
Aurélien
